docs/reference/supported-environments.md

```diff
@@ -33,7 +33,25 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
```

Comment:
Should we explicitly link and state here which setups we support / not support? Basically what we have here: https://docs.camunda.io/docs/next/self-managed/platform-deployment/overview/#deployment-options

Reply:
I added links to the deployment options in my latest commit. We shouldn't document what we don't support. We should focus on what we do support.